src/rmw.c

```diff
@@ -709,3 +717,28 @@ create_undo_file (st_removed *removals, st_removed *removals_head)
     open_err (undo_path, __func__);
 }
 
+/*!
+ * Verify with the user about an action. The prompt produced will be
+ * "Do you want to continue? (y/n): ". If the user provides "y",
+ * then 1 is returned, else 0 is returned.
+ * @return bool
+ */
+bool
+user_verify (void)
+{
+  printf("Do you want to continue? (y/n): ");
```

Comment:
@Jammyjamjamman I think you can simply this function:

```
	char answer = getchar ();
	return (answer == 'y' || answer == 'Y');
```

Reply:
If you write it so that `getchar()` is called only once, any characters after the first char is left behind in the stdout buffer (including the newline when the user verifies their response). This can cause errors later if the buffer is used again. The way I've written it means the buffer is flushed and the user MUST type exactly 'y' (e.g. 'yfhfpwfhew' is not valid even though the first char is 'y').

Side note, I notice your version accepts 'Y' or 'y'. Would you prefer this over just accepting 'y' (which is what my code does)?